### DISCUSSION TITLE
oh-my-posh don' t recognize nerd fonts

### DISCUSSION THREAD

**Discussion by yumetsuro** [JanDeDobbeleer/oh-my-posh#1579] [Troubleshoot]
Hi, i just installed oh my posh and selected a bunch of themes to try it. I installed also FiraCode Nerd fonts
But when i load some themes, they have some strange character, and sometimes annoying lines that ruin the theme.  What could it be?

OS: Linux
Here a screenshot:
![Screenshot from 2022-01-14 20-34-11](https://user-images.githubusercontent.com/21275101/149574656-067cbd8d-815c-4308-99bf-7c2e70279ec6.png)

**Comment by JanDeDobbeleer** [comment]
That's [OSC99](https://ohmyposh.dev/docs/config-overview#general-settings) and your terminal that can't handle it. Disabling that will render everything correctly. Also, oh-my-posh can't influence the font of your terminal, it only provides a set of characters to render. If you see strange lines, that could imply the font size needs a bit of adjustment, or the terminal settings.

**Reply by yumetsuro** [reply]
How do i disable OSC99? 
And i tried to change font size, type of font, and in terminal setting the only thing that affect it line is the cell spacing, that is already at the minimum, here an example with cell spacing set to max:
![Screenshot from 2022-01-14 21-33-19](https://user-images.githubusercontent.com/21275101/149581512-d7d80c99-f2c2-45f3-9700-76b92abfdd3b.png)

**Reply by JanDeDobbeleer** [reply]
@U-n-Own you need to adjust the configuration, see the [docs](https://ohmyposh.dev/docs/config-overview) for more information on that (I assume you downloaded an existing theme to get started?)
